smrf/tests/smrf_test_case.py

```diff
@@ -52,6 +52,10 @@ def dist_variables(self):
     def base_config(self):
         return self.base_config_copy()
 
+    @property
+    def smrf_instance(self):
```

Comment:
Good call, we do use the SMRF instance a lot! This is returning a new instance each time correct? I cannot be certain that there is some less than optimal coding that may change the SMRF instance, so I wouldn't want that to propagate to other tests.

Reply:
Correct, a new instance each time. Had the same thinking as you and wanted to ensure that there is a 'fresh' instance and less chance for test failures due to 'leftovers' from previous test cases.